dnf+

```diff
@@ -0,0 +1,19 @@
+---
+tags: [ packaging ]
+---
+# To install a package:
+dnf install <package>
+
+# To find packages matching <phrase>:
+dnf search <phrase>
+
+# To find which package provides an executable:
+dnf provides <executable>
```

Comment:
Multiple fixes available: please chose one of them.

```suggestion
# Find which package provides a specific executable:
dnf provides "<executable>"
```

Almost the same as above.

```suggestion
# Find which package provides one of specific executables:
dnf provides "<executable1 executable2 ...>"
```

- Almost the same as above.
- Optional change: if `dnf` can accept multiple executables at once for `provides` command then u can use this fix.